packages/cloudflare/src/utils/parser.ts

```diff
@@ -184,7 +186,8 @@ export function getDOBindings(): Record<
 		string,
 		{ scriptName?: string | undefined; unsafeUniqueKey?: string | undefined; className: string }
 	>;
-	for (const binding of rawConfig?.durable_objects.bindings) {
+	const bindings = rawConfig?.durable_objects.bindings;
+	for (const binding of bindings) {
```

Comment:
Using an optional chaining operator inside a for loop is very unsafe, because it can evaluate to `undefined` during the runtime

https://biomejs.dev/linter/rules/use-optional-chain/